### DISCUSSION TITLE
Videos not appearing

### DISCUSSION THREAD

**Discussion by keijiota** [jspsych/jsPsych#3232] [Q&A]
Dear community,

I use a plugin video-keyboard-response and ask people make a judgment after watching a video clip. But sometimes I see that videos fail to play and it leaves a blank background. It produces an error message below on a console. 

rollbar.min.js:1 Rollbar: DOMException: The play() request was interrupted because video-only background media was paused to save power. https://goo.gl/LdLk22
Uncaught (in promise) DOMException: The play() request was interrupted because video-only background media was paused to save power. https://goo.gl/LdLk22

This happens regardless of which web browser I use and whether the pc is save power mode. 
I understand there is always a risk of not being able to play videos. But I need to take a record of it when it happens. Would you please let me know how to get this error response and write it on the data file of the excel sheet?

Many thanks as always.
Keiji

**Comment by kurokida** [comment]
Hi, Keiji-san,

I have never faced this problem and cannot directly answer your question. I recommend you to check wheter you properly preload the movie files.

Best regards,
Daiichiro

**Reply by keijiota** [reply]
I think there is an error on yes loading files. Appreciate your feedback!

**Reply by kurokida** [reply]
Since automatic file loading may not work, it may be better to load the files manually using [the preload plugin](https://www.jspsych.org/7.3/plugins/preload/)
